backend/src/Controller.ts

```diff
@@ -108,6 +108,7 @@ class Controller {
 
   interfaces = async () => {
     await lan.getInterfaces()
+    environment.checkOob()
```

Comment:
The out of band check method should be part of the lan class, not the environment I don't think. Environment should be static information. Also if this is an asynchronous call so it should be a promise, otherwise the data won't be available when the interfaces emit.

Reply:
I didn't realize the lan class was being used this way. Makes more sense now.  Looks like theres a few other things that should move later like privateIP and hostname.  I've moved the data and logic there.  the emit was being done at the end of the function so it did work but I like the promise better.  Updated to use promise.

Reply:
ty!